public/client.js

```diff
@@ -242,19 +243,23 @@ window.addEventListener("load", function () {
 
     //load order for screen 1
     addEntity(new Background(1));
-    addEntity(new Floor(100,100,500,500));
-    addEntity(new Wall(350,100,20,500));
-    addEntity(new Wall(350,600,20,500));
-    addEntity(new Wall(100,350,500,20));
-    addEntity(new Wall(600,350,500,20));
+    addEntity(new Floor(100, 100, 500, 500));
+    addEntity(new Wall(350, 100, 20, 500));
+    addEntity(new Wall(350, 600, 20, 500));
+    addEntity(new Wall(100, 350, 500, 20));
+    addEntity(new Wall(600, 350, 500, 20));
     addEntity(player);
 
     //load order for screen 3
     addEntity(new Background(3));
     addEntity(new TitleCard(3));
-    addEntity(new TitleButton(a.width/2 - 200, 400, 'Connect', 'ssh', function () {socket.emit('join', roomsAvailable[0]);}));
-    addEntity(new TitleButton((a.width/2) - 200, 440, 'Our Creators', 'blame'));
-    addEntity(new TitleButton(a.width/2 - 200, 480, 'Internal Documentation', 'man'));
+    addEntity(new TitleButton(a.width / 2 - 200, 400, 'Connect', 'ssh', function () {
+        if (roomsAvailable && roomsAvailable["1"]) {
+            socket.emit('join', roomsAvailable["1"]);
```

Comment:
I think @KingCole22  changed it so you can just socket.emit('join');

Can you confirm this Morgo?